corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -520,16 +520,16 @@ def hard_delete_forms(domain, form_ids, delete_attachments=True):
         return deleted_count
 
     @staticmethod
-    def archive_form(form, user_id=None):
+    def archive_form(form, user_id):
         from corehq.form_processor.change_publishers import publish_form_saved
-        FormAccessorSQL._archive_unarchive_form(form, user_id, True)
+        FormAccessorSQL()._archive_unarchive_form(form, user_id, archive=True)
         form.state = XFormInstanceSQL.ARCHIVED
         publish_form_saved(form)
 
     @staticmethod
-    def unarchive_form(form, user_id=None):
+    def unarchive_form(form, user_id):
         from corehq.form_processor.change_publishers import publish_form_saved
-        FormAccessorSQL._archive_unarchive_form(form, user_id, False)
+        FormAccessorSQL()._archive_unarchive_form(form, user_id, archive=False)
```

Comment:
can remove the `()` after the method is changed back to static (same above)

Reply:
https://github.com/dimagi/commcare-hq/pull/22470/commits/543f0c30a88142f1452ba2676affc6cca8aa2ddd